R/integration.R

```diff
@@ -145,7 +145,7 @@ runFastMNN = function(sobj, batch) {
 
 	expr <- sobj@assays$RNA@data
 
-	sce <- fastMNN(expr, batch = batch)
+	sce <- fastMNN(expr, batch = sobj@meta.data[[batch]])
```

Comment:
this works directly on a Seurat object, rather than a SingleCellExperiment object? I thought this was bioconductor rather than Seurat focused? @lazappi?

Reply:
The function will take a SingleCellExperiment or just a matrix. I thought a matrix was easier than making the object when we don't need it for anything else so I just extracted that (see Line 146).